programs/util.h

```diff
@@ -287,14 +292,24 @@ UTIL_STATIC int UTIL_isRegFile(const char* infilename);
 UTIL_STATIC int UTIL_setFileStat(const char *filename, stat_t *statbuf)
 {
     int res = 0;
+#if defined(_WIN32) || (PLATFORM_POSIX_VERSION < 200809L)
```

Comment:
minor: coding style : 

You could create a dedicated block `{ ... }` below
where these variables are declared, alongside the code using them.
Since they are not used outside of the block, it's also great to end their lifetime at block boundary.

The benefit is that changes are well localized in one place.

Reply:
The downside is that it will throw C89 warnings.

Reply:
Not if you create a nested block.
Declaration of variables are still allowed in `C89` at the top of any sub-nested block.

Reply:
ehhh nvm. This project passes -std=c99. Made the change.

Reply:
```
void foo() {
  int a;
  a = 1;
  int b;
  b = 2;
}
```

Fails ANSI C.

```
void foo() {
  int a;
  a = 1;
  {
    int b;
    b = 2;
  }
}
```

This passes, because all declarations are at the beginning of an enclosing scope.

Reply:
Used this suggestion.